rosdep/python.yaml

```diff
@@ -1641,6 +1641,7 @@ python-opencv:
   ubuntu:
     saucy: [python-opencv]
     trusty: [python-opencv]
+    xenial: [python-opencv]
```

Comment:
can you use the collapsed version: `ubuntu: [python-opencv]` this way we won't have to modify the key for yakkety, zesty, artful and the following distros ? (noting that starting with artful this will be opencv3 and not opencv2 bindings)